stage1/init/common/kvm_mount.go

```diff
@@ -55,6 +56,12 @@ const (
 	stage1MntDir = "/mnt/"
 )
 
+var log *rktlog.Logger
+
+func init() {
+	log = rktlog.New(os.Stderr, "kvm mount", false)
```

Comment:
this is never used for printing errors in this context so it's really moot point here. I'll default to  debug here though.